content/news/XXXX-XX-XX-mixxx-2-4-0-features.md

```diff
@@ -0,0 +1,85 @@
+title: Mixxx 2.4 Release
+authors: Jörg Wartenberg
+status: draft
+tags: release, 2.4
+comments: yes
+
+
+We are proud to announce Mixxx 2.4, which will be released NEXT WEEK, a new major version of our free open source DJ software. It took about two years since the last release due to the large number of new features and changes under the hood.
+
+Here you can find some highlights of the Mixxx 2.4 features:
```

Comment:
Perhaps we could shorten this a bit to avoid overly relying on addressing the reader with "you":

```suggestion
The 2.4 release includes the following highlights:
```